### DISCUSSION TITLE
Release v2.2.2

### DISCUSSION THREAD

**Discussion by androidacy-user** [Androidacy/MagiskModuleManager#51] [General]
Find out what's new:

[Read more 👉](https://www.androidacy.com/module-manager-update-version-2-2-2-patch-release/?utm_source=GitHub&utm_medium=social&utm_campaign=social2web)

Downloads are linked in the release notes above. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Androidacy/MagiskModuleManager/releases/tag/v2.2.2'>Release v2.2.2</a>.</em>

**Comment by osm0sis** [comment]
Time to rename the builds to e.g. AMM-2.2.2-arm64-v8a-release.apk? 🙂

**Reply by androidacy-user** [reply]
> Time to rename the builds to e.g. AMM-2.2.2-arm64-v8a-release.apk? 🙂

At some point probably, lol. For now, we will keep the name to ease the transition a bit.

**Comment by Masamune3210** [comment]
What is the point of even doing releases on Github if the release notes and the download links are on an entirely different site? Genuine question.

**Reply by androidacy-user** [reply]
> What is the point of even doing releases on Github if the release notes and the download links are on an entirely different site? Genuine question.

Some people may choose to subscribe to GitHub for notifications, and parts of the app still link to GitHub releases for now.

**Reply by now0clock** [reply]
Why not put the .apk files & release notes with the releases here on GitHub like every other Android application?

**Reply by androidacy-user** [reply]
> Why not put the .apk files & release notes with the releases here on GitHub like every other Android application?

Most of the apps in the play store would like a word, along with quite a few not.

We prefer to keep an end to end experience to ensure stability, integrity, and for internal statistics, revenue and our internal build pipeline. Uploading to GitHub requires maintaining multiple release sources, and is harder to pull/replace should the need arise.

We will continue to post releases with information only until the app is fully retooled to use our API for updates.

**Reply by Masamune3210** [reply]
If you are going to come up with an excuse, either make sure its a good one or just....say what you mean. You want to keep it on your site so that you can get ad revenue from people visiting the site and so that you can collect statistics and other metadata easier. Saying that you are keeping it off of github for stability and integrity issues is the most backward thing I have ever heard. It's no harder to pull a release off of github than it is to pull it off of your site, you literally just delete the release, and poof, it's gone. Most CI infrastructures enable easy uploading to github so that's not much of an excuse either. 

If you want people to stop whining so much, stop trying to treat your users like children who don't understand what is going on.

**Reply by androidacy-user** [reply]
> If you are going to come up with an excuse, either make sure its a good one or just....say what you mean. You want to keep it on your site so that you can get ad revenue from people visiting the site and so that you can collect statistics and other metadata easier. Saying that you are keeping it off of github for stability and integrity issues is the most backward thing I have ever heard. It's no harder to pull a release off of github than it is to pull it off of your site, you literally just delete the release, and poof, it's gone. Most CI infrastructures enable easy uploading to github so that's not much of an excuse either. 
> 
> If you want people to stop whining so much, stop trying to treat your users like children who don't understand what is going on. 

We use our own proprietary tooling for CI and related but frankly that's not your business. We have our reasons whether you like them or not. 

If you don't want to be treated like whiny children, perhaps stop acting like it. A few ads never hurt anyone and constantly expecting something for nothing is not going to get you anywhere.

If you have constructive feedback regarding downloads, use the form and button on the app and website. Otherwise, I'm locking this.

**Comment by idanny76** [comment]
> What is the point of even doing releases on Github if the release notes and the download links are on an entirely different site? Genuine question.

I agree... Makes no sense
